python/graphscope/proto/op_def.proto

```diff
@@ -64,28 +57,17 @@ message OpResult {
   // if success, store the metrics. (e.g. how many seconds used, memory bytes...)
   string metrics = 3;
```

Comment:
I suggest we wrap an object-like meta, including:
-  status-code?/error-code?
- details: string additional info?
- metrics: time/memory 
- has_large_results(?)

Reply:
Good suggestion~